src/components/ComboChart.tsx

```diff
@@ -52,15 +54,24 @@ export function ComboChart(props: IComboChartProps): JSX.Element {
     }
 
     const { primaryMeasures = [], secondaryMeasures = [] } = clonedProps;
```

Comment:
whole this preparation of measures starts to be quite big, I would like to separate it into extra function(s)